src/pages/api/users/[user-id].ts

```diff
@@ -1,21 +1,54 @@
+import {
+  getSession,
+  UserProfile,
+  withApiAuthRequired,
+} from "@auth0/nextjs-auth0";
 import { User } from "@prisma/client";
 import { withSentry } from "@sentry/nextjs";
 import { NextApiHandler } from "next";
 
 import { getUser, updateUser } from "../../../services/user";
 
+interface Session {
+  user: UserProfile;
+}
+
 const handler: NextApiHandler = async (req, res) => {
   const userId = req.query["user-id"] as string;
+  const { user: sessionUser } = getSession(req, res) as Session;
+
+  if (!userId) {
+    res.status(422).json({
+      error: "missing_param",
+      description: "query param user-id is missing",
+    });
+
+    return;
+  }
+
```

Comment:
Not sure this param can be missing. The route is `src/pages/api/users/[user-id].ts` and Next should not call the handler if we hit `/api/users//` etc.